src/resources/views/crud/buttons/quick.blade.php

```diff
@@ -23,6 +23,24 @@
         $wrapper['href'] = ($wrapper['href'])($entry, $crud);
     }
     $wrapper['class'] = $wrapper['class'] ?? $defaultClass;
+    //if ajax enabled
+    $ajax_enabled = $button->meta['ajax'] ?? false;
```

Comment:
Maybe we add an additional check here to avoid syntax errors ? 
```suggestion
    $buttonAjaxConfiguration= $button->meta['ajax'] ?? false;
    if($buttonAjaxConfiguration && !is_array($buttonAjaxConfiguration))   {
        $buttonAjaxConfiguration = false;
     }
```

Reply:
I've also renamed the variable, I think it's more clear what it holds. 👍

Reply:
Hum .. sure you are right. It does not need to be an array, disregard that part. 

 Let's just rename the variable, either what I proposed, or `quickAjaxConfiguration` or something similar.

Reply:
Done!